dataactcore/local_secrets_example.yml

```diff
@@ -34,8 +34,17 @@ broker:
     # value if use_aws is true.
     aws_bucket: s3-bucket
 
-    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
+    # Miscellaneous buckets
+    # Location for various current data sources (location, tas, gtas, submission schedule, etc.)
     sf_133_bucket: s3-bucket
+    # Location for older data sources (historic DUNS, historic FABS, historic FPDS,
+    archive_bucket: s3-bucket
+    # Location for other current data sources (FSRS)
+    data_extracts_bucket: s3-bucket
+    # Location for other current data sources (Program Activity)
+    data_sources_bucket: s3-bucket
+    # Location for deleted records from the FPDS atom feed
+    fpds_delete_bucket: s3-bucket
```

Comment:
Are these repeated buckets in both secrets and the config intentional? Shouldn't they be in one place or the other?

Reply:
I was following the same pattern with the aws config (`use_aws`, `aws_region`, `aws_bucket`). I can keep them in the secrets?

Reply:
I think we have a lot of bad, old entries in the files. I don't know if it needs to be secrets or can just be config since it's just bucket names but either way is fine I think.